### PR TITLE
Add Forms

### PR DESCRIPTION
**Pull request by gids-dadi**
In the this project, I added forms for 
 - Creating post
 - Creating comments
 - Both with likes

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id] || params[:id])
```
Comment (Munsa1) [inline app/controllers/likes_controller.rb]: - [ ] Kindly add seed data to your schema, so that upon initialization, the forms can be availabe to the user

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @gids-dadi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — COMMENTED** [review]
### Hello @gids-dadi 
### _This is an additional review as a result of the second opinion request._ 

- Creating a seed file is not a requirement.
- But the project has one problem. I highly recommend fixing it because this is a requirement.
- [ ]  I can't create a post when clicking on the submit button. 

- However,  the last code reviewer did not mention the problem so if you ask for another review in your Dashboard the next Code Reviewer will mark it as approved right away






___
_**Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away**_



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @gids-dadi  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**